services/src/modules/plugins/index.ts

```diff
@@ -39,3 +40,23 @@ export async function buildArgumentInjectionGlobals(): Promise<Record<string, un
     .filter(p => p.addArgumentInjectionGlobals)
     .reduce(async (acc, cur) => ({ ...acc, ...(await cur.addArgumentInjectionGlobals!()) }), {});
 }
+
+export async function transformResourcesUpdates(
```

Comment:
You didn't actually call this in `loadPlugins`

Reply:
It shouldn't be called in `loadPlugins` it's called in `fetchAndValidate`.